src/views/home/post/text-type/ActivityStreamsTextType.vue

```diff
@@ -1,12 +1,24 @@
 <template>
-  <div v-html="data.content"></div>
+  <div>
```

Comment:
that shouldn't be only for text types but maybe for all types? Because every type can have attachments...

Reply:
You're right. The attachment component is unnecessary here. The attachments are already rendered in the right place, in Post.vue